src/components/side-bar/base-components.js

```diff
@@ -73,9 +73,9 @@ class Anchors extends React.PureComponent {
     data.forEach(anchorObj => {
       const id = _.get(anchorObj, 'id', '')
       const label = _.get(anchorObj, 'label', '')
+      const show = _.get(anchorObj, 'show', true)
 
-      // id and label are not empty string
-      if (id && label) {
+      if (show && id && label) {
```

Comment:
If the `show` property is used for the indicator to display,  why not `if (show)`?

Reply:
Sure, I think `show` is enough here.

Updated.